docs/user-documentation/collections.md

```diff
@@ -1,6 +1,7 @@
 Collections are groups of repository content that can be described themselves. Repository content is aggregated in a collection by setting the `field_member_of`
 entity reference field to point to a collection. Generally speaking, anything can behave as a collection, and anything with `field_member_of` can be a member of a collection.
-However, collections are 'officially' designated as such by applying the "Collection" taxonomy term to an item's `field_model` field.  By default, Islandora 8 has a context configured to handle collections by looking for that term.
+However, collections are 'officially' designated as such by applying the "Collection" taxonomy term to an item's `field_model` field.  
+By default, Islandora 8 provides a context configured to display a list of child resource node teasers when the current resource node uses that term.
```

Comment:
I think it's more accurate to say that Islandora "provides a "Collection" taxonomy term available in `field_model` that can cause this node to be treated as a Collection. For example, Islandora Defaults provides a Context that displays a list of child resource node teasers when the current..."